CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.15.0 (TBD)
+## 1.15.0 (2024-04-24)
 
 ### New Features
 
```

Comment:
Added support for registering udfs and stored procedure to local testing.
->
Added support for registering UDFs and stored procedures to local testing.

Reply:
Added distributed tracing using open telemetry apis
->
 Added distributed tracing using open telemetry APIs

Reply:
Re: "UDAF client support is ready for public preview. Please stay tuned for the Snowflake announcement of UDAF public preview." Please discuss with PM if they want forward-looking statements in the documentation.

Reply:
cc @sfc-gh-jfreeberg ^

Reply:
Yeah, the server changes will be available on May 6th. Public blog post on May 8. I'm OK with this language